### DISCUSSION TITLE
Aquifer types in PorousFlow

### DISCUSSION THREAD

**Discussion by AmandaLug** [idaholab/moose#17053] [Q&A General] [answered]
Dear MOOSE,

How can I add layers to my PorousFlow models? I'm using the mesh system of MOOSE. And, how can I layer up different types of aquifers, for example: A unconfined aquifer followed by a confined aquifer and then a leaky aquifer and so on?

Thanks!
Amanda

**Comment by AmandaLug** [comment]
So figured how to add layers in the groundwater documentation... Sorry! But I still have the aquifer types question :confused:.

Amanda

**Comment by aeslaughter** [comment]
@WilkAndy Can you help out here?

**Comment by WilkAndy** [comment, marked as answer]
In hypothetical models, this is reasonably easy.  You can see some models at https://mooseframework.inl.gov/modules/porous_flow/groundwater_models.html where aquitards and aquifers have been defined:

1. Create your mesh so that it has different "blocks" (also called "subdomains")
2. Prescribe different hydraulic properties, such as porosity and permeability, to those blocks

That's all.  I'm not sure exactly what your "leaky aquifer" is... if you mean just something that's half-way between an aquifer and an aquitard, then it'll just need "half-way" permeability, but if you mean something containing a drain (or equivalent) you'll need

3. Apply suitable boundary conditions or point-sinks so appropriate layers to simulate regional groundwater flow or flow to a sink.

The problems come when you're trying to do more sophisticated models with topography and hydrostratigraphy that have complicated geometry.  Then (1) is often quite difficult, in fact (1) often takes the largest fraction of numerical-model-building time in my experience.  But then (2) and (3) follow pretty quickly.

**Reply by AmandaLug** [reply]
Thanks. I understand, I got how to add different layer in simple geometries. 
With my question I meant different types of aquifer confinement. I was thinking about it and probably the best way to confine a aquifer is to add thin low-conductivity layers at the top and bottom of the aquifer that I wanna confine. If that makes sense for you...

Creating different geometries sounds complicated! Do you have or know further references to read about it?
 
Amanda

**Reply by WilkAndy** [reply]
You are correct about the low-conductivity layers.  In fact, that is basically the definition of a "confined aquifer".  The aquitards need not be thin in reality - it is fairly common to have a 10m thick clay band appearing within a sedimentary lithology, and that layer could have conductivity lower than 1E-3 milliDarcy (=1E-18 m^2), meaning water takes about 1000 years to naturally percolate through it.

**Reply by WilkAndy** [reply]
I don't know of any good references that describe building model meshes.  Often they are so customised.  The meshes are often built on layers coming from geology-modelling software such as Leapfrog3D, GoCad, Petrel, ArcGIS.

a

**Reply by AmandaLug** [reply]
This is great. Thanks Andy for your help!

**Comment by WilkAndy** [comment]
Welcome to MOOSE, @AmandaLug .   You'll probably find it hard-going initially, but after a time you'll be familiar with all the concepts and it'll become second-nature.  Are you working at a university?  Are you a groundwater hydrogeologist, or something else?

**Reply by aeslaughter** [reply]
This can help you understand developing using MOOSE: https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/index.html

**Reply by AmandaLug** [reply]
Thanks again! 

Yeah is taking some time to get familiar to PorousFlow. I'm a geophysics, I work for a small consultancy company in Brazil. But I enjoy learning new things!. I came across MOOSE last year in a seminar. 

I'm not really experienced in modeling, I appreciate your complete response. 

Amanda
